Build is failing after an FSComp.txt change

There's a new error when changing the generated errors:
```
4>Microsoft.DotNet.XliffTasks.targets(84,5): Error  :
    'xlf/FSComp.txt.cs.xlf' is out-of-date with '/Users/eugene/Developer/fsharp/artifacts/obj/FSharp.Compiler.Service/Debug/netstandard2.0/FSComp.resx'.
    Run `msbuild /t:UpdateXlf` to update .xlf files or set UpdateXlfOnBuild=true to update them on every build,
    but note that it is strongly discouraged to set UpdateXlfOnBuild=true in official/CI build environments as they should not modify source code during the build.
```

It blocks the development flow and requires you to manually do something that usually is only needed just before opening a PR, or could be done completely automatic on non-CI builds.

I can figure out the needed command, but I feel that it's going to be an issue for some other contributors, especially newcomers.

This is what I get if I try to run the command suggested in the error message:
```
eugene@Eugenes-Air:fsharp$ msbuild /t:UpdateXlf
Microsoft (R) Build Engine version 16.10.1 for Mono
Copyright (C) Microsoft Corporation. All rights reserved.

MSBUILD : error MSB1011: Specify which project or solution file to use because this folder contains more than one project or solution file.
```

The suggestion to set `<UpdateXlfOnBuild>true</UpdateXlfOnBuild>` in the project file doesn't fix the error either.

--- a/Directory.Build.targets
+++ b/Directory.Build.targets
@@ -13,7 +13,4 @@
     <PackageReference Include="xunit.runner.visualstudio" Version="$(XUnitRunnerVersion)" />
     <PackageReference Include="NunitXml.TestLogger" Version="$(NunitXmlTestLoggerVersion)" />
   </ItemGroup>
- <ItemGroup>
-    <PackageReference Condition="'$(BUILDING_USING_DOTNET)' == 'true'" Include="Microsoft.DotNet.XliffTasks" Version="$(MicrosoftDotNetXliffTasksVersion)" PrivateAssets="all" />  
- </ItemGroup>
 </Project>

--- a/eng/Versions.props
+++ b/eng/Versions.props
@@ -192,7 +192,6 @@
     <XUnitRunnerVersion>2.4.2</XUnitRunnerVersion>
     <FluentAssertionsVersion>5.10.3</FluentAssertionsVersion>
     <HumanizerCoreVersion>2.2.0</HumanizerCoreVersion>
-    <MicrosoftDotNetXliffTasksVersion>1.0.0-beta.23475.1</MicrosoftDotNetXliffTasksVersion>
     <!-- MIBC profile packages -->
     <optimizationwindows_ntx64MIBCRuntimeVersion>1.0.0-prerelease.23614.4</optimizationwindows_ntx64MIBCRuntimeVersion>
     <optimizationwindows_ntx86MIBCRuntimeVersion>1.0.0-prerelease.23614.4</optimizationwindows_ntx86MIBCRuntimeVersion>

